test/unit/modules/signatures.js

```diff
@@ -14,66 +14,226 @@
 
 'use strict';
 
+const application = require('../../common/application');
+const transactionTypes = require('../../../helpers/transaction_types');
+const ApiError = require('../../../helpers/api_error');
+
 /* eslint-disable mocha/no-pending-tests */
+/* eslint-disable no-unused-vars */
 describe('signatures', () => {
+	let library;
+
+	before(done => {
+		application.init(
+			{ sandbox: { name: 'lisk_test_modules_signatures' } },
+			(err, scope) => {
+				library = scope;
+				done(err);
+			}
+		);
+	});
+
 	describe('isLoaded', () => {
-		it('should return true if modules exists');
+		it('should return true if modules exists', done => {
+			library.rewiredModules.signatures.__set__('modules', {
+				accounts: library.modules.accounts,
+				transactions: library.modules.transactions,
+				transport: library.modules.transport,
+			});
+			expect(library.modules.signatures.isLoaded()).to.be.true;
+			done();
+		});
 
-		it('should return true if modules does not exist');
+		it('should return true if modules does not exist', done => {
+			const revert = library.rewiredModules.signatures.__set__(
+				'modules',
+				undefined
+			);
+			expect(library.modules.signatures.isLoaded()).to.be.false;
+			revert();
+			done();
+		});
 	});
 
 	describe('onBind', () => {
+		let privateModules;
+		let signatureLogicSpy;
+
+		before(done => {
+			privateModules = library.rewiredModules.signatures.__get__('modules');
+			signatureLogicSpy = sinonSandbox.spy(
+				library.rewiredModules.signatures.__get__('__private').assetTypes[
+					transactionTypes.SIGNATURE
+				],
+				'bind'
+			);
+			library.modules.signatures.onBind(library.modules);
+			done();
+		});
+
 		describe('modules', () => {
-			it('should assign accounts');
+			it('should assign accounts', done => {
+				expect(privateModules).to.have.property(
+					'accounts',
+					library.modules.accounts
+				);
+				done();
+			});
 
-			it('should assign transactions');
+			it('should assign transactions', done => {
+				expect(privateModules).to.have.property(
+					'transactions',
+					library.modules.transactions
+				);
+				done();
+			});
 
-			it('should assign transport');
+			it('should assign transport', done => {
+				expect(privateModules).to.have.property(
+					'transport',
+					library.modules.transport
+				);
+				done();
```

Comment:
The three tests above are not async.. So please avoid using `done`. Prefer sync function when it's not async.

Reply:
Done